contracts/UFragmentsPolicy.sol

```diff
@@ -98,43 +113,42 @@ contract UFragmentsPolicy is Ownable {
 
         uint256 supplyAfterRebase = uFrags.rebase(epoch, supplyDelta);
         assert(supplyAfterRebase <= MAX_SUPPLY);
-        emit LogRebase(epoch, exchangeRate, volume, supplyDelta);
+        emit LogRebase(epoch, exchangeRate, cpi, supplyDelta);
     }
 
     /**
-     * @notice Sets the reference to the market oracle.
-     * @param marketOracle_ The address of the market oracle contract.
+     * @notice Sets the reference to the CPI oracle.
+     * @param cpiOracle_ The address of the cpi oracle contract.
      */
-    function setMarketOracle(IMarketOracle marketOracle_)
+    function setCpiOracle(IOracle cpiOracle_)
         external
         onlyOwner
     {
-        marketOracle = marketOracle_;
+        cpiOracle = cpiOracle_;
     }
 
     /**
-     * @notice Sets the deviation threshold. If the exchange rate given by the market
-     *         oracle is within this absolute distance from the target, then no supply
-     *         modifications are made. RATE_DECIMALS fixed point number.
-     * @param deviationThreshold_ The new exchange rate threshold.
+     * @notice Sets the reference to the market oracle.
+     * @param marketOracle_ The address of the market oracle contract.
      */
-    function setDeviationThreshold(uint256 deviationThreshold_)
+    function setMarketOracle(IOracle marketOracle_)
         external
         onlyOwner
     {
-        deviationThreshold = deviationThreshold_;
+        marketOracle = marketOracle_;
     }
 
     /**
-     * @notice Sets the minimum volume. During rebase, the volume must be at least this value before
-     * any supply adjustment is made.
-     * @param minimumVolume_ The new minimum volume, measured in 24hr market Token Volume.
+     * @notice Sets the deviation threshold percentage. If the exchange rate given by the market
+     *         oracle is within this percentage distance from the targetRate, then no supply
+     *         modifications are made. RATE_DECIMALS fixed point number.
+     * @param deviationThresholdPerc_ The new exchange rate threshold percentage.
      */
-    function setMinimumVolume(uint256 minimumVolume_)
+    function setDeviationThresholdPerc(uint256 deviationThresholdPerc_)
         external
         onlyOwner
     {
-        minimumVolume = minimumVolume_;
+        deviationThresholdPerc = deviationThresholdPerc_;
```

Comment:
Something to think about.

`deviationThreshold` is cacluated as follows.

```
deviationThreshold = targetRate.mul(deviationThresholdPerc).div(100)
	.div(10 ** RATE_DECIMALS);
```

Should we add a require statement here to ensure that `deviationThresholdPerc` doesn't cause the multiplication `targetRate * deviationThresholdPerc` to overflow?

Reply:
Yes, however, the question is do we want to add a require here to prevent rebase from failing due to the overflow?

Reply:
We can't check this if the `targetRate` isn't stored.  Closing this 👍